tests/mbt/driver/mbt_test.go

```diff
@@ -520,8 +520,6 @@ func RunItfTrace(t *testing.T, path string) {
 		for _, consumerChainID := range actualRunningConsumerChainIDs {
```

Comment:
We do not use MBT anymore, right? If so, why do we "touch" those files here?

Reply:
To get rid of compilation errors. Maybe it's time to remove the files completely and not have the dead code anymore, but maybe this should be a separate PR